src/main/java/com/powsybl/openloadflow/network/LfAction.java

```diff
@@ -181,10 +202,4 @@ public void apply() {
             branch.getPiModel().setTapPosition(newTapPosition);
         }
     }
-
-    private static void checkBranch(LfBranch branch, String branchId) {
```

Comment:
So, in that case, no log to the user? And ok no exception.

Reply:
I removed the check at lfnetwork level but it would be better to keep it on iidm network

Reply:
There is nothing at iidm network I think.

Reply:
Yes i will add it in this pr

Reply:
I added to check of actions at IIDM network level and at LfNetwork level we can suppose that if not found it is in another component. what are we supposed to do in that case? Maybe we new status added in next core release, we can have the status "no impact" for this operator strategy result?

Also for IIDM check, it is relevant to throw an exception if an action does not exist in the network ? this is what we are doing for contingencies but maybe we should only warm the user that this contingency or action is invalid and discarded?